LoopKitHostedTests.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Needs to be pushed into a `LoopKitHostedTests` directory along with all of the tests.

Reply:
Should mirror the `LoopKitTests` directory and structure, I think.

Reply:
Huh...Xcode created this file, not sure why it was put in this directory location.  I'll fix it.